op-node/rollup/derive/engine_queue.go

```diff
@@ -412,6 +431,26 @@ func (eq *EngineQueue) tryUpdateEngine(ctx context.Context) error {
 	return nil
 }
 
+// checkNewPayloadStatus checks returned status of engine_newPayloadV1 request for next unsafe payload.
+// It returns true if the status is acceptable.
+func (eq *EngineQueue) checkNewPayloadStatus(status eth.ExecutePayloadStatus) bool {
+	if eq.syncCfg.EngineP2PEnabled {
+		// Allow ACCEPTED and SYNCING if engine P2P sync is enabled
```

Comment:
```suggestion
		// Allow SYNCING and ACCEPTED if engine P2P sync is enabled
```
following the code order.